packages/story-editor/src/components/library/panes/text/karma/textSets.cuj.karma.js

```diff
@@ -249,10 +248,12 @@ describe('CUJ: Text Sets (Text and Shape Combinations): Using Text Sets', () =>
       // First hover text set to trigger image generation
       await fixture.events.mouse.moveRel(textSets[1], 10, 10);
 
-      await fixture.events.sleep(800);
+      await fixture.events.sleep(1500);
       // Then click the text set
       await fixture.events.click(textSets[1]);
 
+      await fixture.events.sleep(500);
+
       await waitFor(
```

Comment:
💭  Is this sleep necessary before the `waitFor`?

Reply:
Who knows, it's just a "[Sleep check](https://github.com/GoogleForCreators/web-stories-wp/pull/10261/commits/b6e33322429d1e4ff908ae6e85b794c7081bd633)", I still have to complete my todo list:
![image](https://user-images.githubusercontent.com/21036224/152871443-148726b2-30a3-4fdd-88b7-ea2bdf79306b.png)

Looks like one(?) of those does the job.